src/task1.py

```diff
@@ -0,0 +1,20 @@
+# Question One
+def nameValidator(name):
+    try:
+        s = name.partition(' ')
+        if(not s[0].isalpha() or not s[1].isspace() or not s[-1].isalpha()):
```

Comment:
Best to store them in variables like firstname and lastname, otherwise its difficult to track which is which